table.csv

```diff
@@ -123,6 +123,7 @@ garlic32,                       multiaddr,      0x01bf,         draft,      I2P
 tls,                            multiaddr,      0x01c0,         draft,
 sni,                            multiaddr,      0x01c1,         draft,      Server Name Indication RFC 6066 § 3
 noise,                          multiaddr,      0x01c6,         draft,
+shs,                            multiaddr,      0x01c8,         draft,      Secure Scuttlebut - Secret Handshake Stream
```

Comment:
```suggestion
shs,                            multiaddr,      0x01c8,         draft,      Secure Scuttlebut - Secret Handshake Stream
```

The description in this file helps but would you be open to making the name itself more descriptive?

Reply:
`Scuttlebut` -> `Scuttlebutt`